src/main/java/io/bonitoo/qa/plugin/sample/SamplePlugin.java

```diff
@@ -83,8 +86,29 @@ public SamplePlugin(PluginProperties props, SampleConfig config) {
     this.id = config.getId();
     this.topic = config.getTopic();
     this.items = new HashMap<>();
+    // TODO better support for arrays of items
```

Comment:
what kind of better support, or is this resolved?

Reply:
This was already addressed in changes below this line.  Forgot to remove todo comment.